src/extension.ts

```diff
@@ -1650,8 +1635,8 @@ const waitForRunningPod = (name, callback) => {
 };
 
 function exists(kind, name, handler) {
-    //eslint-disable-next-line no-unused-vars
-    kubectl.invoke('get ' + kind + ' ' + name, (result) => {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Can probably get rid of this eslint directive?

Reply:
👍